browser/src/control/Control.JSDialogBuilder.js

```diff
@@ -2515,6 +2515,23 @@ L.Control.JSDialogBuilder = L.Control.extend({
 		$(control.label).unbind('click');
 
 		if (!builder.map.isLockedItem(data)) {
+			//Allow "Press+Hold" To Move Faster for only "next" and "previous" related commands
+			if(data.id && (data.id.includes('next') || data.id.includes('prev'))) {
```

Comment:
Widget implementation should never contain any hardcoded ids.
Instead we would to handle "properties" of the widget.
So please put some property into `data` and based on that use new functionality.
Like in data `{... presAndHold: true; ... }`  then here  `if (data.pressAndHold) { ...} `

Reply:
Thanks for pointing that out! I’ve updated the code to use properties from the data object instead of hardcoded IDs. The pressAndHold property is now used to control the functionality as you suggested.